### PR TITLE
Use the ::class notation

### PR DESCRIPTION
**Pull request by nWidart**
Use the ::class notation for class references in app/ folder.
Similar to #3403

### CODE REVIEW

**Comment by RomainLanz** [comment]
Do we need the slash at the beginning of each line?

**Comment by taylorotwell** [comment]
Yes since we're in a namespace.

**Comment by RomainLanz** [comment]
Correct. Thanks for this answer.

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Kernel extends ConsoleKernel
      * @var array
      */
     protected $commands = [
-        'App\Console\Commands\Inspire',
+        \App\Console\Commands\Inspire::class,
```
Comment (GrahamCampbell) [inline app/Console/Kernel.php]: Prefer if we imported this.

File: app/Exceptions/Handler.php [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Handler extends ExceptionHandler
      * @var array
      */
     protected $dontReport = [
-        'Symfony\Component\HttpKernel\Exception\HttpException',
+        \Symfony\Component\HttpKernel\Exception\HttpException::class,
```
Comment (GrahamCampbell) [inline app/Exceptions/Handler.php]: Prefer if we imported this.

File: app/Console/Kernel.php (hunk 1 above)
Comment (nWidart) [inline app/Console/Kernel.php]: What if we have 30 commands ? :smiley: 
That was your argument on the other PR, not having tons of imports at top of file.

**Comment by rtablada** [comment]
Local app ::class will be relative or you could import the namespace (I know namespace imports in PHP aren't ideal). So that brings it down to a shorter length and a single import.

**Pull request merged**